CMakeLists.txt

```diff
@@ -458,7 +458,6 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
     is_sdk_requested(ANDROID swift_build_android)
     if(swift_build_android AND ${swift_can_crosscompile_stdlib})
       configure_sdk_unix(ANDROID "Android" "android" "android" "armv7" "armv7-none-linux-androideabi")
-      set(SWIFT_SDK_ANDROID_PATH "${SWIFT_ANDROID_SDK_PATH}")
```

Comment:
Whoops! 😅  Thanks as always, @practicalswift. The Android-related Python changes LGTM, too.